lucene/CHANGES.txt

```diff
@@ -204,7 +204,8 @@ Improvements
 * LUCENE-9446: In BooleanQuery rewrite, always remove MatchAllDocsQuery filter clauses
   when possible. (Julie Tibshirani)
 
-* LUCENE-9501: Improve how Asserting* test classes handle singleton doc values.
+* LUCENE-9501: Improve coverage for Asserting* test classes: make sure to handle singleton doc
+  values, and sometimes wrap Scorers by default. (Julie Tibshirani)
```

Comment:
I'm not sure I'd understand what "sometimes wrap scorers" would mean as a user reading the changelog. Maybe say something along the lines of "sometimes exercise the `Weight#scorer` API rather than `Weight#bulkScorer` for top-level queries".

Reply:
This is a good point, I updated the changelog entry.